tracer/src/Datadog.Trace/AppSec/Waf/rule-set.json

```diff
@@ -297,53 +294,54 @@
               }
             ],
             "list": [
-              ".htaccess",
-              ".htdigest",
-              ".htpasswd",
-              ".addressbook",
-              ".aptitude/config",
-              ".bash_config",
-              ".bash_history",
-              ".bash_logout",
-              ".bash_profile",
-              ".bashrc",
+              "/.htaccess",
+              "/.htdigest",
+              "/.htpasswd",
+              "/.addressbook",
+              "/.aptitude/config",
+              "/.bash_config",
+              "/.bash_history",
+              "/.bash_logout",
+              "/.bash_profile",
+              "/.bashrc",
               ".cache/notify-osd.log",
               ".config/odesk/odesk team.conf",
```

Comment:
is this correct? Some of these don't have the `/` prefix?

Reply:
Yes, this change is meant to cut down on false positives where some patterns (like `.git`) could appear in email adresses (or similar). That was because we were looking for file access pattern without trying to contextualise the match (whether we were matching a file path or any random string).
For those patterns and those patterns only, we're adding `/` before the pattern to increase our confidence that we're matching a path, rather than an out-of-context string. That's not very elegant but the WAF operator we're using there isn't very flexible.

Reply:
Great, thanks 🙂